modules/realvuBidAdapter.js

```diff
@@ -0,0 +1,64 @@
+import adaptermanager from 'src/adaptermanager';
+
+var utils = require('src/utils.js');
```

Comment:
Prefer `const` for `require` statements and anywhere else in the module they can be used in place of `var`